src/components/CategoriesNav/index.js

```diff
@@ -92,6 +92,31 @@ const CategoriesNav = ({ emojiListRef }) => {
             />
           );
         })}
+        {customGroups !== [] &&
```

Comment:
This will *always* be `true`, even if customGroups is an empty array.
![image](https://user-images.githubusercontent.com/11255103/151859603-553fead2-df3c-410b-8ed4-1d66858e8d3e.png)

You probably want something like this:
```suggestion
        {!(Array.isArray(customGroups) && customGroups.length)  ? null :
```

Reply:
Oops! Fixed - commit 2088794380580356e07facab253b63a41d777d7e